tests/test_environs.py

```diff
@@ -407,6 +407,17 @@ def test_read_env_return_path(self, env):
         env_path = str(HERE / ".env")
         assert path == env_path
 
+    def test_read_env_return_path_with_dotenv_on_working_dir(self, env):
+        working_dir = pathlib.Path(os.getcwd())
+        dotenv_path = working_dir / ".env"
+        open(dotenv_path, "x")  # create .env on working dir
```

Comment:
This will result in a failure if the assertion fails and the test is re-run. I'll push up a fix for this shortly